x11-utils/lightdm/BUILD

```diff
@@ -1,10 +1,4 @@
 OPTS+=" --disable-static \
         --disable-tests" &&
 
-default_build &&
-
-install -m 755 $SCRIPT_DIRECTORY/Xsession /etc/lightdm/Xsession &&
-
-# PolicyKit
-install -dm 750 -g 102 /usr/share/polkit-1/rules.d &&
-install -m 644 $SCRIPT_DIRECTORY/lightdm.rules /usr/share/polkit-1/rules.d/lightdm.rules
```

Comment:
@rtlanceroad what was the reason to remove the polkit rules? It was used to be control some functions like controlling upower etc (reboot, shutdown, etc). Does it have other means to deal with that now?